src/Entity/UserConsentInterface.php

```diff
@@ -13,6 +13,21 @@
  */
 interface UserConsentInterface extends ContentEntityInterface, EntityChangedInterface, EntityOwnerInterface {
 
+  /**
+   * The user only visits agreement page.
+   */
+  const STATE_UNDECIED = 0;
```

Comment:
@OH17L this is a typo. Should be `STATE_UNDECIDED`

Reply:
@bramtenhove, you are right. I will fix it.